src/api/app/controllers/webui/user_controller.rb

```diff
@@ -22,7 +22,15 @@ def show
     @role_titles = @displayed_user.roles.global.pluck(:title)
     @account_edit_link = CONFIG['proxy_auth_account_page']
 
-    switch_to_webui2
+    return unless switch_to_webui2
+    @last_day = Time.zone.today
+
+    # take the last 52 weeks
+    @first_day = @last_day - (52 * 7)
```

Comment:
Isn't there some `52.weeks.ago` helper from rails? :thinking:

Reply:
.ago might be useful here, but in the long run we want to have this code also for previous years. So we need some kind of delta anyway

Reply:
Anyway, please use the `.weeks.ago` instead of calculating this yourself.

Reply:
`weeks.ago` is a method of DateTime. We would have to do `weeks.ago.to_date`

Reply:
perhaps `- 52.weeks` is sufficient ?